packages/p2p/src/components/buy-sell/buy-sell-row.jsx

```diff
@@ -8,8 +8,9 @@ import { buy_sell } from 'Constants/buy-sell';
 import { Localize, localize } from 'Components/i18next';
 import UserAvatar from 'Components/user/user-avatar';
 import { useStores } from 'Stores';
-import './buy-sell-row.scss';
+import StarRating from '../star-rating/star-rating.jsx';
```

Comment:
Could you make this an absolute import?
```suggestion
import StarRating from 'Components/star-rating';
```